Makefile

```diff
@@ -34,25 +35,12 @@ tools:
 	
 .PHONY: test
 test:
-	go test ./... -covermode=atomic -coverpkg=./... -v
+	go test ./... -v $(TESTOPTS)
 
-test-cover:
-	go test -covermode=atomic -coverpkg=./... -coverprofile=coverage.out ./...
-
-.PHONY: coverage-html
-cover-html:
-	go tool cover -html=coverage.out
-
-.PHONY:coverage
-coverage:test-cover
-	@{ \
-	set -e ;\
-	REQ_COV=15.5 ;\
-	TEST_COV=$$(go tool cover -func coverage.out | grep -F total | awk '{print substr($$3, 1, length($$3)-1)}' ) ;\
-	if (( $$(echo "$$REQ_COV > $$TEST_COV" |bc -l) )); then echo "Error: Code Coverage is less"; exit 1 ;\
-	else echo "Code Coverage Test Passed"; exit 0; fi ;\
-	}
-	@rm -rf coverage.out
+.PHONY: coverage
+coverage:
+	chmod 755 hack/codecov.sh
```

Comment:
This shouldn't be done in the Makefile. Please perform the locally and change the permissions of `hack/codecov.sh` file to be executable by others.

Reply:
Yes I removed that. Thanks